ui/main/src/app/modules/usercard/usercard.component.ts

```diff
@@ -178,7 +178,8 @@ export class UserCardComponent implements OnDestroy, OnInit {
                         this.messageForm.get('process').disable();
                         this.messageForm.get('state').setValue(this.cardToEdit.card.state);
                         this.messageForm.get('startDate').setValue(getDateTimeNgbFromMoment(moment(this.cardToEdit.card.startDate)));
-                        this.messageForm.get('endDate').setValue(getDateTimeNgbFromMoment(moment(this.cardToEdit.card.endDate)));
+                        if (this.cardToEdit.card.endDate || this.endDateVisible)
+                            this.messageForm.get('endDate').setValue(getDateTimeNgbFromMoment(moment(this.cardToEdit.card.endDate)));
```

Comment:
I'm not sure I understand the point of "|| this.endDateVisible" because if this.cardToEdit.card.endDate is undefined or null and this.endDateVisible is true, it will cause the value to be set with getDateTimeNgbFromMoment(moment(null)). What does moment(null) return ?

Reply:
It will return current datetime.

Reply:
Clever ;-)